Tests/sdk_release/create_release.py

```diff
@@ -0,0 +1,106 @@
+import requests
+import json
+import sys
+import argparse
+import re
+import urllib3
+from distutils.util import strtobool
+from Tests.scripts.utils.log_util import install_logging
+from Tests.scripts.utils import logging_wrapper as logging
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+# regex to split the changelog line to 3 pieces- description, pr number and url
+CHANGELOG_REGEX = re.compile(r"^(.*) \[(#\d+)\]\((http.*)\)")
+
+
+def get_changelog_text(release_branch_name, text_format='markdown'):
+    # get release changelog file
+    url = f"https://raw.githubusercontent.com/demisto/demisto-sdk/{release_branch_name}/CHANGELOG.md"
+    response = requests.request("GET", url, verify=False)
+    if response.status_code != requests.codes.ok:
+        logging.error(f'Failed to get the CHANGELOG.md file from branch {release_branch_name}')
+        logging.error(response.text)
+        sys.exit(1)
+    file_text = response.text
+    release_changes = (
+        file_text.split(f"## {release_branch_name}\n")[1].split("\n\n")[0].split("\n")
+    )
+
+    # Converting release changes to markdown links
+    releases = []
+    for change in release_changes:
+        try:
+            # Ignoring the mypy error because the regex must match
+            change_parts = CHANGELOG_REGEX.match(change).groups()  # type: ignore[union-attr]
```

Comment:
```suggestion
            description, pr_number, url = CHANGELOG_REGEX.match(change).groups()  # type: ignore[union-attr]
```
would make it more readable.